sys/shell/commands/sc_sx127x.c

```diff
@@ -0,0 +1,586 @@
+/*
```

Comment:
#7370 will cause a vast simplification of `sc_netif.c` anyway. Let's see if we can fix it then.